addons/survey/models/survey_survey_template.py

```diff
@@ -0,0 +1,281 @@
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Once a week')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Once a month')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Once a year')
+                        }),
+                        (0, 0, {  # survey.question.answer
+                            'value': _('Never')
+                        })
+                    ]
+                }),
+                (0, 0, { # survey.question
+                    'title': _('How many times did you order products on our website?'),
+                    'question_type': 'numerical_box'
+                }),
+                (0, 0, { # survey.question
+                    'title': _('About our ecommerce'),
+                    'is_page': True,
+                    'question_type': False
+                }),
+                (0, 0, { # survey.question
+                    'title': _('What do you think about our new eCommerce?'),
+                    'question_type': 'matrix',
+                    'suggested_answer_ids': [
+                        (0, 0, { # survey.question.answer
+                            'value': _('Strongly disagree')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Disagree')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Neutral')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Agree')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('Strongly agree')
+                        })
+                    ],
+                    'matrix_subtype': 'simple',
+                    'matrix_row_ids': [
+                        (0, 0, { # survey.question.answer
+                            'value': _('The new layout and design is fresh and up-to-date')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('It is easy to find the product that I want')
+                        }),
+                        (0, 0, { # survey.question.answer
+                            'value': _('The tool to compare the products is useful to make a choice')
+                        })
+                    ]
+                })
+            ]
+        }))
+
+    @api.model
+    def action_load_sample_certification(self):
+        values = {
+            'title': _('Certification'),
+            'certification': True,
+            'access_mode': 'token',
+            'is_time_limited': True,
```

Comment:
Suggestion: 

`'certification_mail_template_id': self.env.ref('survey.mail_template_certification', raise_if_not_found=False) or False`

That way you can keep the same style / code structure as the other ones.

Reply:
If we put the option `raise_if_not_found` to `False`, the `ref` function will return `None` if the record with the given id does not exist. For the `survey_value`, I will have to check that the value returned by the `ref` function is not `None` before accessing the `id` attribute. Otherwise, we will get an error. It would hence be more or less the same.